docs/feature_rgb_matrix.md

```diff
@@ -15,7 +15,20 @@ RGB_MATRIX_ENABLE = yes
 RGB_MATRIX_DRIVER = IS31FL3731
 ```
 
-Configure the hardware via your `config.h`:
+You can use between 1 and 4 IS31FL3731 IC's. Do not specify `DRIVER_ADDR_<N>` defines for IC's that are not present on your keyboard. You can define the following items in `config.h`:
```

Comment:
Needs equivalent docs changes on `feature_led_matrix.md` too.

Reply:
Ah I believe my changed to `feautre_led_matrix.md` got lost in conflict resolutions with 7078d5a5bdf9a639003a124342de567c976182ff .  Added changes back